tests/test_using.py

```diff
@@ -1181,21 +1196,47 @@ class Meta:
             b = 1
 
             # biggest = 'b' if .b > .a else 'a'
-            biggest = factory.Maybe(factory.LazyAttribute(lambda o: o.a < o.b), 'b', 'a')
+            biggest = factory.Maybe(
+                factory.LazyAttribute(lambda o: o.a < o.b), "b", "a"
+            )
             # max_value = .b if .b > .a else .a = max(.a, .b)
             max_value = factory.Maybe(
                 factory.LazyAttribute(lambda o: o.a < o.b),
-                factory.SelfAttribute('b'),
-                factory.SelfAttribute('a'),
+                factory.SelfAttribute("b"),
+                factory.SelfAttribute("a"),
             )
 
         obj_ordered = DummyFactory.build(a=1, b=2)
         obj_equal = DummyFactory.build(a=3, b=3)
         obj_reverse = DummyFactory.build(a=5, b=4)
 
-        self.assertEqual(dict(a=1, b=2, biggest='b', max_value=2, ), obj_ordered.as_dict)
-        self.assertEqual(dict(a=3, b=3, biggest='a', max_value=3, ), obj_equal.as_dict)
-        self.assertEqual(dict(a=5, b=4, biggest='a', max_value=5, ), obj_reverse.as_dict)
+        self.assertEqual(
+            dict(
+                a=1,
+                b=2,
+                biggest="b",
+                max_value=2,
+            ),
+            obj_ordered.as_dict,
+        )
+        self.assertEqual(
+            dict(
+                a=3,
+                b=3,
+                biggest="a",
+                max_value=3,
+            ),
+            obj_equal.as_dict,
+        )
+        self.assertEqual(
+            dict(
+                a=5,
+                b=4,
+                biggest="a",
+                max_value=5,
+            ),
+            obj_reverse.as_dict,
+        )
```

Comment:
The resulting code is much harder to read — it was important to have the `a`, `b`, `biggest` and `max` values being close together for readability.